registry/tests/utils.py

```diff
@@ -68,6 +97,20 @@ def setUp(self):
         sqlalchemy_utils.create_database(self.db_url)
         quilt_server.db.create_all()
 
+        self.email_suffix = '@example.com'
+
+        self.TEST_USER = 'test_user'
+        self.TEST_USER_EMAIL = 'test_user@example.com'
+        self.OTHER_USER = 'share_with'
+        self.OTHER_USER_EMAIL = 'share_with@example.com'
+        _create_or_update_user(self.TEST_USER, email=self.TEST_USER_EMAIL)
```

Comment:
Why do you need "create or update" at all? You just created a new database; it's not gonna have any users, right?